### PR TITLE
[SDK Update] Added Payment Method screen

### PR DESCRIPTION
**Pull request by mpetrenco**
# Changelog:
- Added the iOS Payment Method Screen to the React Native SDK

### CODE REVIEW

File: JudoPay.tsx [hunk 1]
```diff
@@ -32,7 +38,26 @@ class JudoPay {
...
-    private generateParameters(
+    public invokePaymentMethodScreen(
+        mode: JudoTransactionMode,
+        configuration: JudoConfiguration
+    ): Promise<JudoResponse | null> {
+
+        const params = this.generatePaymentMethodParameters(mode, configuration);
+        const judoPay = NativeModules.RNJudo;
+
+        return new Promise((resolve, reject) => {
```
Comment (stpopa) [inline JudoPay.tsx:49]: can we avoid nesting Promises and instead use `async / await`?

File: JudoPay.tsx [hunk 2]
```diff
@@ -32,7 +38,26 @@ class JudoPay {
         });
     }
 
-    private generateParameters(
+    public invokePaymentMethodScreen(
+        mode: JudoTransactionMode,
+        configuration: JudoConfiguration
+    ): Promise<JudoResponse | null> {
```
Comment (stpopa) [inline JudoPay.tsx:44]: `Promise<JudoResponse | null>` can we just `Promise<JudoResponse>` ?

Comment (AndreiGalkin) [reply JudoPay.tsx:44]: It can be null if response won't be that type as expected.

File: JudoPay.tsx [hunk 3]
```diff
@@ -44,6 +69,19 @@ class JudoPay {
             'configuration': configuration,
         }
     }
+
+    private generatePaymentMethodParameters(
+        mode: JudoTransactionMode,
+        configuration: JudoConfiguration
+    ): any {
```
Comment (stpopa) [inline JudoPay.tsx:76]: `any` -> `Record<string, any>` ?
https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkt

File: JudoPay.tsx (hunk 2 above)
Comment (stpopa) [reply JudoPay.tsx:44]: @AndreiGalkin is it possible to throw an error in case of `response won't be that type as expected` ?

Comment (AndreiGalkin) [reply JudoPay.tsx:44]: I think yes, check for type similarity while parsing or wrap in exception handler

**Pull request merged**